packages/app/src/utils.ts

```diff
@@ -76,3 +76,75 @@ export function sendCommand(frame: HTMLIFrameElement, command: any): Promise<any
 export function getProjectId(medplum: MedplumClient): string {
   return resolveId(medplum.getActiveLogin()?.project) as string;
 }
+
+/**
+ * This function reads the contents of the a JSON object, manipulates the object
+ * to add UUIDs and other modifications, and then exports the
+ * modified JSON object to a file.
+ * @param blob The Blob object that we'll receive from the search query
+ */
+export function getFHIRBundle(entry: any): void {
```

Comment:
I would suggest some tweaks to these utility functions, to make them more broadly useful. 
1. This function to should take in a `Bundle` and output a `Bundle`. Have the stringifying / exporting happen in the calling function
2. rename this to `convertToTransactionBundle`, to clarify that the main purpose of this Bundle is to convert one type of Bundle to another
3. Move the bundle conversion functions to `@medplum/core`, that way they can also be used by the CLI and other tools